code/game/objects/effects/anomalies.dm

```diff
@@ -347,3 +362,72 @@
 	//Damaging the turf
 	if( T && prob(turf_removal_chance) )
 		T.ex_act(ex_act_force)
+
+///An anomaly that lubes, drains blood and spews random reagent smoke
+/obj/effect/anomaly/fluid
+	name = "liquescent anomaly"
+	icon_state = "wetball"
+	density = FALSE
+
+	aSignal = /obj/item/assembly/signaler/anomaly/fluid
+
+	///the type of reagent we use as smoke, color and from the anomaly pad
+	var/reagent_type
+	///the volume of our internal anomaly production
+	var/volume = 10
+
+	///how long our lube lasts in deciseconds
+	var/lube_time = 50
+	///chance we make lube
+	var/lube_chance = 20
+	///lube range
+	var/lube_range
+	///u of blood we drain per suck
+	var/blood_drain = 20
+	///blood drain range
+	var/blood_drain_range = 2
+
+	//chance per process that we smoke
+	var/smoke_chance = 10
+	///range of our smoke
+	var/smoke_range = 5
+	///volume of our smoke. If you increase this, increase volume too or there wont be enough smoke to spew
+	var/smoke_volume = 10
+
+/obj/effect/anomaly/fluid/Initialize()
+	. = ..()
+
+	create_reagents(volume)
+
+	SetReagent(get_random_reagent_id())
+
+/obj/effect/anomaly/fluid/anomalyEffect()
+	..()
```

Comment:
```suggestion
	. = ..()
```

Reply:
It doesn't expect a return value, don't wanna be misleading